src/js/common/schemaform/save-in-progress/FormStartControls.jsx

```diff
@@ -23,7 +23,10 @@ class FormStartControls extends React.Component {
   }
 
   handleLoadPrefill = () => {
-    if (this.props.prefillAvailable) {
+    if (this.props.handleLoadPrefill) {
+      // TODO: Is this how handleLoadPrefill is intended to be used?
+      this.props.handleLoadPrefill();
```

Comment:
For this form we are fetching only if ITF is successful, so each of these conditions are now set up to be mutually exclusive. There aren't any other instances of handleLoadPrefill being used, so this change doesn't require additional changes to existing usages. Is this within the realm of how handleLoadPrefill was envisioned to be used? To allow for the prefill to be called directly if certain conditions are met?

Reply:
:man_shrugging: Sounds like a question for The Great Balboni! (cc: @jbalboni)

Reply:
Well, it wasn't meant to be exposed outside of this component, but it is the method called when starting a new form. I don't think this is the right name for it if it's becoming a prop, though. It should be something like `startNewForm`. 

You could also consider creating a `beforeStartNewForm` prop that returned a promise, and have this component do the logic in `handleLoadPrefill` if the promise resolves successfully. That would keep you from duplicating the `fetchInProgressForm` call.